src/Elastic.Transport/Elastic.Transport.csproj

```diff
@@ -27,6 +27,11 @@
     <JsonSerializerIsReflectionEnabledByDefault>false</JsonSerializerIsReflectionEnabledByDefault>
   </PropertyGroup>
 
+  <!-- 👇 disable the nullable warnings when compiling for .NET Standard 2.0 -->
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'net462'">
+    <NoWarn>$(NoWarn);nullable</NoWarn>
```

Comment:
Is that really required? Can't remember having trouble with these when using modern SDKs to compile the code.